### DISCUSSION TITLE
blog/2023/htb-cyber-apocalypse-ctf-2024-web-writeups/

### DISCUSSION THREAD

**Discussion by giscus** [hash3liZer/khatta#15] [Q&A]
# blog/2023/htb-cyber-apocalypse-ctf-2024-web-writeups/

This blog post contains the writeups for the web challenges that i solved from HTB Cyber Apocalypse 2024

https://blog.shameerkashif.me/blog/2023/htb-cyber-apocalypse-ctf-2024-web-writeups/

**Comment by douglasjsmith404** [comment]
just testing your commenting ...  nice!     struggling with 'webmentions'
